opm/simulators/flow/FlowMain.hpp

```diff
@@ -155,11 +155,6 @@ namespace Opm {
             EWOMS_HIDE_PARAM(TypeTag, NewtonTargetIterations);
             EWOMS_HIDE_PARAM(TypeTag, NewtonVerbose);
             EWOMS_HIDE_PARAM(TypeTag, NewtonWriteConvergence);
-            EWOMS_HIDE_PARAM(TypeTag, EclNewtonSumTolerance);
```

Comment:
question, why removing them?

Reply:
Is it because in the TestTypeTag.hh, they are defined?

Reply:
Because the were defined in the eclnewtonmethod.hh which we are removing. These settings are not used anywhere in flow, they were only ever used by ebos.